home/models.py

```diff
@@ -17,34 +19,36 @@ def __str__(self):
         return f"{self.name} | {self.email}"
 
     class Meta:
-        ordering = ('-created',)
-        constraints = [
-            models.UniqueConstraint(fields=['email', 'device_id'], name='Unique user + device')
-        ]
+        ordering = ("-created",)
+
 
 # Event model
 class DailyWalk(models.Model):
+    # Identifier
+    event_id = models.CharField(max_length=250, primary_key=True)
+    # Walk meta
     date = models.DateField()
     steps = models.IntegerField()
```

Comment:
In addition to steps, we're collecting and displaying distance in the app, so we might as well store it in the backend as well? A float/double value...

Reply:
Yes! It'd definitely be great if whatever pieces of information you're saving makes its way to the backend as well. I'll add this in.